atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/internalschema/TransactionSchemaManager.java

```diff
@@ -87,9 +92,16 @@ public boolean tryInstallNewTransactionsSchemaVersion(int newVersion) {
     private InternalSchemaMetadata installNewVersionInMapOrDefault(int newVersion,
             ValueAndBound<InternalSchemaMetadata> valueAndBound) {
         if (!valueAndBound.value().isPresent()) {
+            log.info("Attempting to install a new transactions schema version {}, but no past data was found,"
```

Comment:
Make it a warn, and say this should happen only once on first startup?

Reply:
Yep, done. The wording is unfortunately a little more guarded, because of the possibility of a race condition.